### ISSUE TITLE
Where Calculating the metrics on test sets in your code?

### ISSUE THREAD

**Issue by lingxitong** [JianghaiSCU/Diffusion-Low-Light#20]
Hello,nice work,I can not find the code to cal the metrics(PSNR ↑ SSIM ↑ LPIPS ↓ FID ↓)  in your paper,can you tell me where it exists?thanks

**Comment by JianghaiSCU** [comment]
We use skimage to calculate PSNR and SSIM. LPIPS is calculated based on the official version and FID is calculated based on https://github.com/chaofengc/IQA-PyTorch.

**Comment by lingxitong** [comment]
thanks,another question is I do not find LOLv1 test set,I only find our485 and eval15 folder for train and eval,can you tell me where to get the LOLv1 test set to get the metrix in your paper

**Comment by JianghaiSCU** [comment]
eval15 is the test set.
